src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -315,10 +360,23 @@ const Notebooks = _.flow(
             icon('plus-circle', { style: { marginTop: '0.5rem' }, size: 21 })
           ])
         ]),
+        getConfig().enableGalaxy && h(Fragment, [
```

Comment:
This should also be conditional on being in the Anvil site skin, for now.

Reply:
For ease of testing and development, we are posing that this happens as part of SATURN-1808